backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -0,0 +1,285 @@
+                .AsNoTracking()
+                .OrderBy(s => s.SortOrder)
+                .ThenBy(s => s.Name)
+                .Where(s => EF.Functions.Like(s.Workflow, $"%{workflow}%"));
+
+            return status.ToArray();
+        }
+
+        /// <summary>
+        /// Get the project for the specified 'projectNumber'.
+        /// Will not return sensitive properties unless the user has the `sensitive-view` claim and belongs to the owning agency.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <exception type="KeyNotFoundException">Entity does not exist in the datasource.</exception>
+        /// <returns></returns>
+        public Project Get(string projectNumber)
+        {
+            if (String.IsNullOrWhiteSpace(projectNumber)) throw new ArgumentException("Argument cannot be null, empty or whitespace.", nameof(projectNumber));
+            this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
+
+            // Check if user has the ability to view sensitive properties.
+            var userAgencies = this.User.GetAgencies();
+            var viewSensitive = this.User.HasPermission(Permissions.SensitiveView);
+            var isAdmin = this.User.HasPermission(Permissions.AdminProperties);
+
+            var project = this.Context.Projects
+                .Include(p => p.Status)
+                .Include(p => p.TierLevel)
+                .Include(p => p.Agency)
+                .Include(p => p.Agency.Parent)
+                .Include(p => p.Tasks)
+                .Include(p => p.Tasks).ThenInclude(t => t.Task)
+                .Include(p => p.Properties)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Evaluations)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Fiscals)
+                .Include(p => p.Properties).ThenInclude(b => b.Building)
+                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Evaluations)
+                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Fiscals)
+                .AsNoTracking()
+                .FirstOrDefault(p => p.ProjectNumber == projectNumber &&
+                    (isAdmin || userAgencies.Contains(p.AgencyId))) ?? throw new KeyNotFoundException();
+
+            // Remove any sensitive properties from the results if the user is not allowed to view them.
+            if (!viewSensitive)
+            {
+                project?.Properties.RemoveAll(p => p.Parcel.IsSensitive);
+                project?.Properties.RemoveAll(p => p.Building.IsSensitive);
+            }
+            return project;
+        }
+
+        /// <summary>
+        /// Add the specified project to the datasource.
+        /// </summary>
+        /// <param name="project"></param>
+        /// <returns></returns>
+        public Project Add(Project project)
```

Comment:
Do we care if a property is in more than one project?

Reply:
We do, but there is an issue we'll need to solve which is when a property has been in a prior project (approved/denied/transferred) and now it is part of another.  Which means we don't want a property to be part of multiple 'open' projects.